src/core/tailer/tailer.go

```diff
@@ -95,6 +101,28 @@ func NewPatternTailer(file string, patterns map[string]string) (*PatternTailer,
 	return &PatternTailer{t, gc}, nil
 }
 
+func NewLTSVTailer(file string) (*LTSVTailer, error) {
+	ltsvParseString := func(line string) map[string]string {
```

Comment:
Can you move this out into its own function?

Reply:
@dhurley I could move out the parse function outside.